src/pages/popup/components/TogglePaymentsButton.tsx

```diff
@@ -3,14 +3,10 @@ import { Power } from '@/pages/shared/components/Icons';
 import { cn } from '@/shared/helpers';
 import { useMessage } from '@/popup/lib/context';
 import { dispatch } from '@/popup/lib/store';
+import { usePopupState } from '@/popup/lib/store';
```

Comment:
```suggestion
import { dispatch, usePopupState } from '@/popup/lib/store';
```

Reply:
should we add eslint to enforce a single import statement for modules with the same path? [no-duplicate-imports](https://eslint.org/docs/latest/rules/no-duplicate-imports)